Automattic/wp-calypso
# Migrate language picker styles webpack; Disallow lexical declarations on a default clause of a switch

#### Changes proposed in this Pull Request

- Split `Language Picker` stylesheet into two smaller stylesheets
- Import these two stylesheets with webpack. Details on https://github.com/Automattic/wp-calypso/issues/27515
- Fix `no-case-declarations` linter warning for default clause in a switch

#### Testing instructions

- Visit `/devdocs/design/language-picker` on the live branch available below
- Test the feature here and ensure the design/visual appearance looks the same as this page's view on `master` branch / before this PR
- Can also test on a language switcher for a site, on site settings, or on Calypso interface settings here `/me/account`

<img width="1489" alt="Screenshot 2019-04-02 at 11 33 48" src="https://user-images.githubusercontent.com/18581859/55379592-3446d700-553b-11e9-8b38-760f992f4fb0.png">
